app/controllers/plants_controller.rb

```diff
@@ -10,6 +10,7 @@ def index
 
   # GET /plants/1 or /plants/1.json
   def show
+    @pagy, @watering_logs = pagy(@plant.watering_logs.order(created_at: :desc), items: 3)
```

Comment:
Do we need `items: 3`? 🤔 I think it does nothing. If you want to adjust the number of records per page the correct option is `limit: 3`. 

💡 you can also do something like `limit: params[:limit] || 3` - then FE can decide on how many records to display on a single page. It is a common pattern, e.g. in e-commerce stores. But we don't use it in our case so there is no need for that 👍

Reply:
Yes, this is not needed. It was for testing purposes and I forgot to clear this.
Typically this behavior is handled by pagy.rb initializer.

![obraz](https://github.com/user-attachments/assets/53823b58-d30b-4b11-83b9-b36a0b389ab2)